app/src/main/java/com/gatech/asl_dictionary/Search.kt

```diff
@@ -55,6 +75,28 @@ fun Search(navigationToSecondScreen: (String) -> Unit) {
                 }
             }
         }
+
+
+        var inputText by remember { mutableStateOf("") }
```

Comment:
TODO: check if inputText is in Signs.json and do whatever clickButton(signEntry) does

Reply:
Yeah. I was going to implement this separately. Actually im working on it now.

Reply:
Yeah. I was going to implement this separately. Actually im working on it now.